utility/make_private_namespace.py

```diff
@@ -0,0 +1,45 @@
+""" Create a namespace """
+import os
+import subprocess
+import shlex
+from utility import templating
+from ocs.utils import create_oc_resource
+
+
+def make_private_namespace(name, region='us-east-2'):
+    """
+    Create a namespace.
+
+    Args:
+        name (str): namespace to be created
+        region (str): aws region.  Defaults to us-east-2
+
+    Returns:
+        No return value.  The new namespace should exist, and a directory in
+        /tmp with the namespace's name should contain the private.yaml file
+        that created this namespace.
+
+    """
+    oc_cmd = "oc get namespace"
+    process = subprocess.Popen(shlex.split(oc_cmd), stdout=subprocess.PIPE)
+    output, error = process.communicate()
+    if error:
+        print(f"Error executing: {oc_cmd}")
+        return
+    prev_ns = [x for x in output.decode().split('\n') if x.startswith(f"{name} ")]
+    if prev_ns:
+        print(f"{name} already exists")
+        return
+    env_data = dict()
+    env_data['platform'] = 'AWS'
```

Comment:
there is no 'platform' in the private.yaml

Reply:
Done.  Accidentally left this in.